libraries/stdlib/samples/test/samples/misc/controlFlow.kt

```diff
@@ -15,9 +15,14 @@ class ControlFlow {
         repeat(3) {
             println("Hello")
         }
+        
+        // greets with a count
+        repeat(3) { count ->
```

Comment:
Better to name the lambda parameter `iteration`/`index` or something like that, otherwise it makes an impression that `count` would be equal to 3.

Reply:
Done